src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -532,6 +534,27 @@ public Iterable<Article> getArticleFromSearch(String locale, String searchTerm,
         return complete(submit(req("GET", tmpl), handleList(Article.class, "results")));
     }
 
+    public MediaRecord getMediaByName(String name) {
```

Comment:
IMO we should keep SDK simple. Your method includes business logic that is specific to the current task implementation. If we will decide to change the flow then we will need to update the SDK.
This method should take all possible parameters (search term, sorting, and pagination) and return the list as is. The actual pagination loop and logic should be in the connector.